WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModelTests.swift

```diff
@@ -131,6 +136,92 @@ final class StoreOnboardingViewModelTests: XCTestCase {
         XCTAssertEqual(sut.tasksForDisplay[1].task.type, .launchStore)
     }
 
+    func test_tasksForDisplay_contains_launch_store_task_for_WPCOM_site_under_free_trail() async {
```

Comment:
nit:
```suggestion
    func test_tasksForDisplay_contains_launch_store_task_for_WPCOM_site_under_free_trial() async {
```

Reply:
Done in 1551a9c1830ad6c206f564cb3290c353d515c419